src/osd/ReplicatedPG.cc

```diff
@@ -1310,6 +1315,54 @@ void ReplicatedPG::do_pg_op(OpRequestRef op)
   delete filter;
 }
 
+int ReplicatedPG::do_scrub_ls(MOSDOp *m, OSDOp *osd_op)
+{
```

Comment:
pool.info.supports_omap() is about the PGBackend.  All objectstore implementations much support omap, however.  It's perfectly fine to use omap even if the PGBackend doesn't support it (we aren't writing the scrub results using the PGBackend).